### PR TITLE
ref(js): Convert PipelineView to a functional component

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: static/app/views/integrationPipeline/pipelineView.tsx [hunk 1]
```diff
@@ -24,27 +20,27 @@ type Props = {
...
+
+  const [Component, title] = mapping;
+
+  // Set the page title
+  useEffect(() => void (document.title = title), [title]);
+
+  return (
+    <ThemeAndStyleProvider>
+      <Indicators className="indicators-container" />
+      <Component {...props} />
```
Comment (matejminar) [inline static/app/views/integrationPipeline/pipelineView.tsx:41]: Should we also pass `pipelineName` as previously?

Comment (evanpurkhiser) [reply static/app/views/integrationPipeline/pipelineView.tsx:41]: I checked, it wasn't being used.

File: static/app/views/integrationPipeline/pipelineView.tsx [hunk 2]
```diff
@@ -24,27 +20,27 @@ type Props = {
...
+  const mapping = pipelineMapper[pipelineName];
+
+  if (!mapping) {
+    throw new Error(`Invalid pipeline name ${pipelineName}`);
   }
+
+  const [Component, title] = mapping;
+
+  // Set the page title
+  useEffect(() => void (document.title = title), [title]);
```
Comment (markstory) [inline static/app/views/integrationPipeline/pipelineView.tsx:36]: Should we be using `react-document-title` here? Could be done separately.

**Review by markstory — APPROVED** [review]

**Review by priscilawebdev — APPROVED** [review]

File: static/app/views/integrationPipeline/pipelineView.tsx (hunk 2 above)
Comment (evanpurkhiser) [reply static/app/views/integrationPipeline/pipelineView.tsx:36]: Yeah can updated after

**Pull request merged**